### PR TITLE
Add episode 29

### PR DESCRIPTION
**Pull request by bitsondatadev**

### CODE REVIEW

File: _episodes/29.md
Comment (mosabua) [inline _episodes/29.md:24]: code highlighting

File: _episodes/29.md
Comment (mosabua) [inline _episodes/29.md:25]: A whole bunch of

File: _episodes/29.md
Comment (mosabua) [inline _episodes/29.md:40]: make this list shorter and end with "and others" .. and link that to the connectors section of the docs

File: _episodes/29.md
Comment (mosabua) [inline _episodes/29.md:46]: It helps to start by looking at Trino's architecture:

File: _episodes/29.md
Comment (mosabua) [inline _episodes/29.md:55]: plans, and schedules the processing of SQL queries. The queries are submitted by users directly or with connected SQL reporting tools.

File: _episodes/29.md
Comment (mosabua) [inline _episodes/29.md:56]: actually carry out more of the processing by reading data from

File: _episodes/29.md
Comment (mosabua) [inline _episodes/29.md:95]: possible, depending

File: _episodes/29.md
Comment (mosabua) [inline _episodes/29.md:112]: empty line after title

File: _episodes/29.md
Comment (mosabua) [inline _episodes/29.md:110]: need to add the last part ...

workers submitted processed data back to coordinator .. and coordinator provides the results of the query to the user.

File: _episodes/29.md
Comment (mosabua) [inline _episodes/29.md:114]: query processing metadata

File: _episodes/29.md
Comment (mosabua) [inline _episodes/29.md:119]: remove the whole stuff in ()

File: _episodes/29.md
Comment (mosabua) [inline _episodes/29.md:123]: Also .. we might want to mention that this was contributed by Bloomberg (if we are allowed) .. and we definitely should link to the docs for it

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _episodes/29.md
Comment (bitsondatadev) [reply _episodes/29.md:40]: I wanted to keep this here to be thorough and also for the SEO :)

Comment (mosabua) [reply _episodes/29.md:40]: sure .. but still add the "and others" and the link .. you could even make each word a link...

File: _episodes/29.md
Comment (mosabua) [inline _episodes/29.md:37]: remove "also"

File: _episodes/29.md
Comment (mosabua) [inline _episodes/29.md:130]: Add the answer here. .. 

In short, the answer is that the Hive runtime is not required. A lot more information is available in the Intro ...

**Review by mosabua — APPROVED** [review]
Only minimal tweaks.. once those are done .. request review/merge from maintainers.

**Comment by bitsondatadev** [comment]
@hashhar, martin is a bit busy at the moment, Adding him on just in case but would you mind reviewing please?

File: _episodes/29.md
Comment (hashhar) [inline _episodes/29.md:24]:
```suggestion
* `ALTER MATERIALIZED VIEW ... RENAME TO`
```

This renders better than how it looks now.

File: _episodes/29.md
Comment (hashhar) [inline _episodes/29.md:26]: maybe?

```suggestion
* Elasticsearch connector no longer fails if fields with unsupported types exist
```

File: _episodes/29.md
Comment (hashhar) [inline _episodes/29.md:28]:
```suggestion
* Parquet and Avro fixes and improvements
```

File: _episodes/29.md
Comment (hashhar) [reply _episodes/29.md:40]: A link would be nice for sure.

File: _episodes/29.md
Comment (hashhar) [inline _episodes/29.md:53]: types of nodes doesn't care about counts so `a _coordinator_` reads weird. Maybe drop the `a`?

File: _episodes/29.md
Comment (hashhar) [inline _episodes/29.md:67]:
```suggestion
implement the SPI which includes exposing metadata, statistics, data location, and establishing one or 
```

File: _episodes/29.md
Comment (hashhar) [inline _episodes/29.md:45]: The images below this point all appear "before" the text that explains the image which seems like the reverse of what reads naturally. Was this intended? Either way is fine but I personally prefer the images following the text (especially because the paragraphs are small and images close together).

File: _episodes/29.md
Comment (hashhar) [inline _episodes/29.md:102]:
```suggestion
Data is split and distributed across the worker nodes to provide inter-node parallelism.
```

File: _episodes/29.md
Comment (hashhar) [inline _episodes/29.md:109]:
```suggestion
Once this data arrives to the worker node, it is further divided and processed in 
```

File: _episodes/29.md
Comment (hashhar) [inline _episodes/29.md:118]:
```suggestion
[Pull request 8821]({{site.github_repo_url}}/pull/8821) enables Trino cluster 
owners to log query processing metadata by submitting it to an HTTP endpoint. This may be used for usage monitoring and 
alarming, but it might also be used to extract analytics on cluster usage, such 
as tables/column usage metrics.
```

Comment (hashhar) [reply _episodes/29.md:118]: Or `submit query processing metadata to an HTTP endpoint`.

File: _episodes/29.md
Comment (hashhar) [inline _episodes/29.md:121]: maybe?
```suggestion
or over HTTPS. Configuration allows selecting which events should be included.
```

File: _episodes/29.md
Comment (hashhar) [inline _episodes/29.md:123]: Maybe link to `mosiac1`'s GitHub?

File: _episodes/29.md
Comment (hashhar) [inline _episodes/29.md:130]: `week's`?

**Review by hashhar — COMMENTED** [review]
Some comments.

File: _episodes/29.md
Comment (bitsondatadev) [reply _episodes/29.md:45]: Yeah, I actually think this changes depending on the person. I make inferences based on visual queues first then like to read the rationale behind the image after. In certain cases where I think the image is confusing without explanation I may do some explaining to ease the user into an image. I leave this up to personal style and will just leave it as is if that's cool with you.

**Review by hashhar — APPROVED** [review]
LGTM

**Pull request merged**
